packages/redux/src/settings/reducer/__tests__/setting.test.ts

```diff
@@ -0,0 +1,129 @@
+import * as actionTypes from '../../actionTypes.js';
+import * as fromReducer from '../settings.js';
+import { settingsReducer as reducer } from '..//index.js';
+import { toBlackoutError } from '@farfetch/blackout-client';
+import type { SettingState } from '../../types/index.js';
+
+let initialState: SettingState;
+const randomAction = { type: 'this_is_a_random_action' };
+
+describe('setting redux reducer', () => {
+  beforeEach(() => {
+    initialState = fromReducer.INITIAL_STATE;
+  });
+
+  describe('error() reducer', () => {
+    it('should return the initial state', () => {
+      const state = reducer(undefined, randomAction).error;
+
+      expect(state).toBe(initialState.error);
+      expect(state).toBeNull();
+    });
+
+    it('should handle FETCH_SETTING_REQUEST action type', () => {
+      expect(
+        reducer(undefined, {
```

Comment:
Please, use a more appropriate state instead of using `undefined`. This applies to all tests here except the ones that test for the retrieval of the initial state.

Reply:
Done 👍